assets/share_asset.py

```diff
@@ -0,0 +1,115 @@
+import sublime_plugin
+import sublime
+from pieces_os_client import LinkifyApi,Linkify
+
+from .utils import AssetSnapshot
+from .list_assets import PiecesListAssetsCommand
+from .create_asset import PiecesCreateAssetCommand
+from ..settings import PiecesSettings
+from ..auth.auth_user import AuthUser
+
+
+class PiecesShareAssetCommand(sublime_plugin.WindowCommand):
+	def run(self,asset_id,update_sheet=False):
+		self.update_sheet = update_sheet
+		sublime.set_timeout_async(lambda:self.run_async(asset_id))
+
+	def run_async(self,asset_id=None,seed=None):
+		"""
+			You need to either give the seed or the asset_id
+		"""
+		if asset_id:
+			kwargs = {"asset" : AssetSnapshot.assets_snapshot[asset_id]}
+		else:
+			kwargs = {"seed" : seed}
+		user = AuthUser.user_profile
+		if not user:
```

Comment:
instead of preventing these actions here we might want to just connect their allocation

Reply:
Yupp already did that in the line under, prompt to login and he can press on login to open the browser